wrapanapi/ec2.py

```diff
@@ -64,6 +64,19 @@ class EC2System(WrapanapiAPIBaseVM):
         'deleted': ('terminated',),
     }
 
+    stack_states = {
```

Comment:
**Required:**  Should add a comment describing these, and mentioning they can be used with list_stacks().

Reply:
not sure that cross-links are required from one place to another and back